gradle/dependencies.gradle

```diff
@@ -23,7 +23,7 @@ ext {
 
   version = [
       mapboxMapSdk              : '10.18.1',
-      mapboxSdkServices         : '6.15.0',
+      mapboxSdkServices         : '7.1.0',
```

Comment:
mapbox-java 6.15.0 and 7.1.0 are compatible, no breaking changes between them.

Reply:
why are we bumping major version if there're no breaking changes?

Reply:
@VysotskiVadim I believe this was done by a mistake when search team released this https://github.com/mapbox/mapbox-java/releases/tag/v7.0.0.

I think it doesn't make sense to stick v6 and duplicate minor releases for both v6 and v7 given that they're binary compatible. This shouldn't cause any issues.

Reply:
wow 😨